OPERATOR.rst

```diff
@@ -156,28 +156,45 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to the GitLab instances and the GitLab Runners
+on a daily basis. Updates to the Gitlab instance can be quickly checked at the
+``/help`` endpoint for `GitLab dev`_ and `GitLab prod`_. If there are updates
+available, find the new release for the `GitLab Docker Image`_ and update the
+`Gitlab terraform`_ file. When updating the GitLab instance check if there are
+applicable updates to the `GitLab Runner image`_. Before starting the update
+process check applicable `GitLab release notes`_ for any upgrading instructions.
+When upgrading across multiple Gitlab versions follow a supported GitLab
+`upgrade path`_.
+
+Before any changes are applied, have the lead check for any running
+``background migrations`` on the GitLab instances, if there are no migrations
+in progress, stop the instance (do not terminate) and create a snapshot of its
+EBS volume.
```

Comment:
Confusing. This sound like the only the snapshotting is conditional upon the existence of background migrations. In fact, the entire upgrade is. Structure this so that checking the conditions are the first steps in the a enumerated list and describe each branch in the decision tree. Think of this as a [flow chart](https://en.wikipedia.org/wiki/Flowchart) described in plain English.